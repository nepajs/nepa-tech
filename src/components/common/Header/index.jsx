import React from 'react'
import styled from 'styled-components'

const Header = () => <StyledHeader>Header</StyledHeader>

const StyledHeader = styled.header``

export default Header
