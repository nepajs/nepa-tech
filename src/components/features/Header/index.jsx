import React from 'react'
import styled from 'styled-components'
import Link from 'gatsby-link'

const Header = ({ siteTitle }) => (
  <StyledHeader>
    <h1>
      <Link to="/">{siteTitle}</Link>
    </h1>
  </StyledHeader>
)

const StyledHeader = styled.header``

export default Header
