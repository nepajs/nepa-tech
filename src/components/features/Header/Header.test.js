import React from 'react'
import Header from '../Header'

it('renders correctly', () => {
  const tree = shallow(<Header />)
  expect(toJson(tree)).toMatchSnapshot()
})
