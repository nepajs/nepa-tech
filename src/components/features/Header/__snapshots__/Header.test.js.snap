// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.header>
  <h1>
    <GatsbyLink
      to="/"
    />
  </h1>
</styled.header>
`;
